docs/releases/2.2.1.txt

```diff
@@ -18,7 +18,10 @@ Bugfixes
 * Fixed a regression in Django 2.2 that caused a crash when loading the
   template for the technical 500 debug page (:ticket:`30324`).
 
+* Fixed a regression in Django 2.2 that caused a crash when IntegerField
+  limits are callable.
+
 * Fixed crash of ``ordering`` argument in
   :class:`~django.contrib.postgres.aggregates.ArrayAgg` and
   :class:`~django.contrib.postgres.aggregates.StringAgg` when it contains an
-  expression with params (:ticket:`30332`).
+  expression with params (:ticket:`30332`).
```

Comment:
Please ensure that there is a new line at the end of the file, i.e. revert the change that removes it.